### DISCUSSION TITLE
Whats the difference between F0 detections

### DISCUSSION THREAD

**Discussion by Justarandimmboi** [w-okada/voice-changer#1202] [Q&A] [answered]
Just want to know what the difference is between all F0 detection presets

**Comment by AnnoyingFox** [comment, marked as answer]
Here you can have a look at the description of the F0 detection. Basically, depending on the preset it may be more or less accurate detecting your voice or wether it uses the cpu or gpu. In my case, rnvpe gives the best results with noticeable difference.

https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md#f0-det-f0-extractor

**Reply by Justarandimmboi** [reply]
I was looking for that and didn't see it prob cause I scrolled too fast. 

Cheers
